src/server/db/models/github-user.js

```diff
@@ -7,9 +7,32 @@ export default function register(db) {
    *   created_at: creation date
    *   updated_at: update date
    *   last_login_at: last login date
+   *   snaps_added: number of snaps added
+   *   snaps_removed: number of snaps removed
+   *   names_registered: number of snap names registered
```

Comment:
These are just an incremental counters per user? It doesn't matter if I add and remove same snap over and over again, it will bump up the stat?